src/index.ts

```diff
@@ -1756,7 +1758,7 @@ function findNodeIndex(
 }
 
 /* JSX Types */
-import { JSXTypes } from "./jsxTypes";
```

Comment:
This changes the import to only happen at `tsc`-time, and esbuild doesn't treat it as a 'real' import.